src/xunit.performance.api/PerformanceTestConfig.cs

```
@@ -0,0 +1,51 @@
+using System;
+
+
+namespace Microsoft.Xunit.Performance.Api
+{
+    public class PerformanceTestConfig
+    {
+        public PerformanceTestConfig() 
+        {
+            Iterations = 30;
+            TimeoutPerIteration = TimeSpan.FromMilliseconds(20000);
+        }
+
+        public TimeSpan TimeoutPerIteration 
+        {
+            get { return _timeoutPerIteration; }
+
+            set
+            {
+                if (value.TotalMilliseconds > 0) 
+                {
+                    _timeoutPerIteration = value;
+                }
+                else 
+                {
+                    Console.Error.WriteLine("Timeout per iteration should be a positive amount of milliseconds.");
```

I would prefer to throw an exception here.